builtins/amp-img.js

```diff
@@ -56,7 +57,8 @@ export class AmpImg extends BaseElement {
           value => mutations[value] !== undefined);
       this.propagateAttributes(
           attrs, this.img_, /* opt_removeMissingAttrs */ true);
-      this.guaranteeSrcForSrcsetUnsupportedBrowsers_();
+      const srcset = this.element.getAttribute('srcset');
```

Comment:
We could do this inside `guaranteeSrcForSrcsetUnsupportedBrowsers`.

Reply:
Done